ui-admin/src/study/participants/survey/PrintFormModal.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect } from 'react'
+import { Survey as SurveyComponent } from 'survey-react-ui'
+
+import {
+  surveyJSModelFromForm,
+  makeSurveyJsData,
+  waitForImages,
+  Survey,
+  ConsentForm,
+  configureModelForPrint
+} from '@juniper/ui-core'
+
+import { Answer } from 'api/api'
+import Modal from 'react-bootstrap/Modal'
+import { Button } from 'components/forms/Button'
+
+
+type DownloadFormViewProps = {
+    answers: Answer[],
+    resumeData?: string,
+    survey: Survey | ConsentForm,
+    onDismiss: () => void
+}
+
+
+/** renders the form in a fullscreen modal, and pops up the print dialog */
+const PrintFormModal = ({ survey, answers, resumeData, onDismiss }: DownloadFormViewProps) => {
```

Comment:
I wonder if this should be a routable page instead of a modal. The modal might be what's getting in the way of PgUp/PgDn not working. Also, since the modal takes up the whole screen and doesn't look like a modal, I'm tempted to think that the back button should go back to the survey view, not back to wherever I was before opening the survey.

However, I did just instinctively hit `esc` to close the print view and was happy that it worked. Maybe it should stay as a modal but make it just barely less than the full screen so that it looks more like a modal?

Reply:
yeah, I'd like to get more of a sense of how this will be used.  My spidey sense is that this functionality is very temporary -- either admin users will almost never use it, in which case it doesn't matter except as an occasional nice-to-have, or we will have a customer that needs it for all participants (like the Pepper studies that auto-send consent pdfs to various third parties) in which case we'll want to invest in some sort of Nodejs backend that can generate these pdfs server-side

Reply:
good idea on making it routable -- done in latest commit